evalml/objectives/lead_scoring.py

```diff
@@ -43,5 +43,7 @@ def objective_function(self, y_true, y_predicted, X=None):
         profit += self.false_positives * false_positives
 
         profit_per_lead = profit / len(y_true)
+        # penalty if our estimator only predicts 1 output
+        same_class_penalty = (2 - len(set(y_predicted))) * (abs(profit_per_lead) / 2)
```

Comment:
Adding a penalty if our estimator predicts only 1 output. Added this to try and avoid threshold graphs where the `0` or `1` threshold would result in the highest scores

Reply:
Got it.

Why the division by 2?

Reply:
@dsherry the division by 2 was just to make the penalty smaller. There was no real reason beyond that though. I can remove it if it makes it simpler?

Reply:
I think that would make it so that the score of a model that predicts one label is `<= 0` right? Seems like a desirable property!

Reply:
Thanks for explaining. That sounds good to me. Simpler is always good in my book :)